api/envoy/type/grpc/v2/grpc_status.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
```

Comment:
Where is this used or planned to be used?

Reply:
See the linked PR in the description.